services/website/website.service.js

```diff
@@ -67,22 +67,27 @@ export default class Website extends BaseService {
       url,
     },
   ) {
-    let isUp
-    try {
-      const {
-        res: { statusCode },
-      } = await this._request({
-        url,
-        options: {
-          method: 'HEAD',
-        },
-      })
-      // We consider all HTTP status codes below 310 as success.
-      isUp = statusCode < 310
-    } catch (e) {
-      // Catch all errors thrown by the request.
-      isUp = false
-    }
+    let isUp = false
+    // eslint-disable-next-line no-async-promise-executor
+    await new Promise(async (resolve, reject) => {
+      setTimeout(resolve, 3500)
```

Comment:
The only problem with this is eslint errs when a `Promise` executor is provided an async function.

Initially, I've got the idea from [this SO answer](https://stackoverflow.com/a/52059193/2926992).

I've run the test a couple of times to check if async `Promise` executor will err due to (not sure but) some kind of race condition maybe. It was consistent. Still awaiting review though.

Reply:
I think it would make more sense to pass [got](https://github.com/sindresorhus/got) a timeout in the `options` object and catch a `TimeoutError` if thrown to render the "down" badge.

https://github.com/sindresorhus/got/blob/main/documentation/2-options.md#timeout
https://github.com/sindresorhus/got/blob/main/documentation/6-timeout.md

Reply:
Completed.